Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -306,16 +491,1190 @@ script:
+      - il-central-1
+      - mx-central-1
+      - me-south-1
+      - me-central-1
+      - sa-east-1
+    
+    - name: group_id
+      description: 'The ID of the security group.'
+      required: true
+    - name: protocol
+      description: 'The IP protocol: tcp, udp, icmp, or icmpv6. Use -1 to specify all protocols. When used with port and cidr arguments for simple rule revocation.'
+      required: false
+    - name: port
+      description: 'For TCP or UDP: The range of ports to revoke (e.g., "80" or "80-443"). For ICMP: A single integer or range (type-code) representing the ICMP type and code. Use with protocol and cidr for simple rule revocation.'
+      required: false
+    - name: cidr
+      description: 'The IPv4 address range in CIDR format (e.g., "0.0.0.0/0"). Use with protocol and port for simple rule revocation.'
+      required: false
+    - name: ip_permissions
+      description: 'The sets of IP permissions to revoke, in JSON format. Use this for complex rule configurations or when revoking multiple rules. Cannot be used together with protocol/port/cidr arguments.'
+      required: false
+
+  - name: aws-ec2-snapshot-attribute-modify
+    description: 'Adds or removes permission settings for the specified snapshot. Note: snapshots encrypted with the AWS-managed default key (alias/aws/ebs) cannot be shared—use unencrypted snapshots or those encrypted with a customer-managed KMS key to allow permission modifications.'
+    arguments:
+    - name: account_id
+      description: The AWS account ID.
+      required: true
+    - name: region
+      description: The AWS region.
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - us-east-1
+      - us-east-2
+      - us-west-1
+      - us-west-2
+      - af-south-1
+      - ap-east-1
+      - ap-south-2
+      - ap-southeast-3
+      - ap-southeast-5
+      - ap-southeast-4
+      - ap-south-1
+      - ap-northeast-3
+      - ap-northeast-2
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-southeast-7
+      - ap-northeast-1
+      - ca-central-1
+      - ca-west-1
+      - eu-central-1
+      - eu-west-1
+      - eu-west-2
+      - eu-south-1
+      - eu-west-3
+      - eu-south-2
+      - eu-north-1
+      - eu-central-2
+      - il-central-1
+      - mx-central-1
+      - me-south-1
+      - me-central-1
+      - sa-east-1
+    
+    - name: snapshot_id
+      description: 'The ID of the snapshot.'
+      required: true
+    - name: attribute
+      description: 'The snapshot attribute to modify.'
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - 'createVolumePermission'
+    - name: operation_type
+      description: 'The operation to perform.'
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - 'add'
+      - 'remove'
+    - name: user_ids
+      description: 'CSV list of  AWS accounts IDs to add to or remove from the list of users that have permission to create EBS volumes from the snapshot.'
```

```suggestion
    - name: user_ids
      isArray: true
      description: 'A comma-separated list of AWS user account IDs to add to or remove from the list of users permitted to create EBS volumes from the snapshot.'
```
correct me if I'm wrong (account IDs => user account IDs)